lib/pacemaker/pcmk_scheduler.c

```diff
@@ -750,7 +750,7 @@ unpack_cib(xmlNode *cib, unsigned long long flags, pe_working_set_t *data_set)
 
     /* This will zero the entire struct without freeing anything first, so
      * callers should never call pcmk__schedule_actions() with a populated data
-     * set unless pe_flag_have_status is set (i.e. cluster_status() was
+     * set unless pcmk_sched_have_status is set (i.e. cluster_status() was
```

Comment:
crap - the compiler wouldn't find that - so it needs a search in the end for exhaustive replacement :-(

Reply:
Yep, but coccinelle helped a lot :)

Reply:
But it didn't catch the asterisk ;-)